hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestRepairsCommand.java

```diff
@@ -234,6 +234,30 @@ public void testOverwriteHoodieProperties() throws IOException {
     assertEquals(expect, got);
   }
 
+  @Test
+  public void testOverwriteHoodiePropertiesWithInputStreamFix() throws IOException {
+    RepairsCommand command = new RepairsCommand();
+    URL newProps = this.getClass().getClassLoader().getResource("table-config.properties");
```

Comment:
Most of the test items are similiar with `testOverwriteHoodieProperties` ?

Reply:
Yes, their general test logic are similar."

Reply:
What's the difference, can you explain the test prupose?

Reply:
The purpose of the test is to call this method and check whether the inputStream is working properly.

Reply:
> to call this method

Which method is called here, does `testOverwriteHoodieProperties` already cover the code path?

Reply:
When i call the method testOverwriteHoodiePropertiesWithInputStreamFix,they will call method overwriteHoodieProperties,so the changes closed input stream with resource will be tested.

Reply:
So `testOverwriteHoodieProperties` already covers this, can you just take an offline validation instead, we can get rid of these UTs if manual test passes.

Reply:
sorry,I don't understand how to validate,and how to get rid of these UTs,can you help me?

Reply:
> get rid of these UTs

I mean there is no need to add UTs, instead, validate these two commds you have modified offline to make sure it works correctly.

Reply:
OK，understand！